<!DOCTYPE html>
<html lang="en">
  <head>
  <title>Galaxy Europe</title>
  <meta property="og:title" content="" />
  <meta property="og:description" content="" />
  <meta property="og:image" content="/assets/media/galaxy-eu-logo.512.png" />
  <meta name="description" content="The European Galaxy Instance">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">




  <link rel="canonical" href="https://galaxyproject.eu/index-singlecell.html">
  <link rel="shortcut icon" href="/assets/media/galaxy-eu-logo.64.png" type="image/x-icon" />
  <link rel="alternate" type="application/rss+xml" title="Galaxy Europe" href="/feed.xml">

  <link href="/assets/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
  <script src="/assets/js/jquery-3.2.1.slim.min.js" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" crossorigin="anonymous"></script>
  <script src="/assets/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
</head>

  <body>
    <div id="wrap">
      <div id="main">
        <div class="container" id="maincontainer">
          <div class="home">







  <p><img src="/assets/media/logo_single_cell.svg" alt="single cell Galaxy" class="sc-intro-left" /></p>

<h1 class="no_toc" id="welcome-to-the-world-of-single-cell-omics">Welcome to the world of Single Cell Omics</h1>

<p>The Single Cell Omics workbench is a comprehensive set of analysis tools and consolidated workflows. The workbench is based on the <a href="https://galaxyproject.org" target="_blank">Galaxy framework</a>,
which guarantees simple access, easy extension, flexible adaption to personal and security needs, and sophisticated analyses independent of command-line knowledge.</p>

<p>The current implementation comprises more than 20 bioinformatics tools dedicated to different research areas of single cell biology.</p>

<p>This service is a joint project between different groups from the <a href="http://www.earlham.ac.uk" target="_blank">Earlham Institute</a>, the <a href="https://www.ebi.ac.uk" target="_blank">EMBL-EBI</a>, <a href="https://gbcs.embl.de" target="_blank">EMBL</a> the <a href="http://artbio.fr" target="_blank">Sorbonne University</a>, <a href="https://www.petermac.org" target="_blank">Peter MacCallum Cancer Centre</a> and the <a href="https://galaxyproject.eu/freiburg/" target="_blank">University of Freiburg</a>.
The server is part if the European Galaxy server and is maintained by the <a href="https://www.denbi.de/network/rna-bioinformatics-center-rbc" target="_blank">RNA Bioinformatics Center (RBC)</a> as part of <a href="https://www.denbi.de" target="_blank">de.NBI</a> and <a href="http://elixir-europe.org" target="_blank">ELIXIR</a>.</p>

<h1 class="no_toc" id="content">Content</h1>

<ol id="markdown-toc">
  <li><a href="#get-started" id="markdown-toc-get-started">Get started</a></li>
  <li><a href="#training-and-workshops" id="markdown-toc-training-and-workshops">Training and Workshops</a>    <ol>
      <li><a href="#workshops" id="markdown-toc-workshops">Workshops</a></li>
      <li><a href="#material" id="markdown-toc-material">Material</a></li>
    </ol>
  </li>
  <li><a href="#available-tools" id="markdown-toc-available-tools">Available tools</a>    <ol>
      <li><a href="#preprocessing" id="markdown-toc-preprocessing">Preprocessing</a></li>
      <li><a href="#downstream" id="markdown-toc-downstream">Downstream</a></li>
      <li><a href="#suites-mentioned" id="markdown-toc-suites-mentioned">Suites Mentioned</a></li>
    </ol>
  </li>
  <li><a href="#contributors" id="markdown-toc-contributors">Contributors</a></li>
</ol>

<h1 id="get-started">Get started</h1>

<p>Are you new to Galaxy, or returning after a long time, and looking for help to get started? Take <a href="https://singlecell.usegalaxy.eu/tours/core.galaxy_ui" target="_blank">a guided tour</a> through Galaxy’s user interface.</p>

<h1 id="training-and-workshops">Training and Workshops</h1>

<p><strong>Want to learn more about single cell analyses? Check out the following hands-on tutorials from <a href="https://galaxyproject.github.io/training-material/">the Galaxy Training Network</a></strong>, or come to one of our workshops.</p>

<p>We are passionate about training. So we are working in close collaboration with the <a href="https://galaxyproject.org/teach/gtn/" target="_blank">Galaxy Training Network (GTN)</a> to develop training materials of data analyses based on Galaxy <a class="citation" href="#batut2017community">(Batut <i>et al.</i>, 2017)</a>. These materials hosted on the GTN GitHub repository are available online at <a href="https://training.galaxyproject.org" target="_blank">https://training.galaxyproject.org</a>.</p>

<h2 id="workshops">Workshops</h2>

<p>We will be hosting a scRNA-seq workshop <a href="https://usegalaxy-eu.github.io/event/2020-01-20-GalaxyWS_scrna_FR/plain.html">this March</a>. Please register whilst spots are still open!</p>

<h2 id="material">Material</h2>

<table class="table table-striped">
  <thead>
    <tr>
      <th>Lesson</th>
      <th>Slides</th>
      <th>Hands-on</th>
      <th>Input dataset</th>
      <th>Workflows</th>
      <th>Galaxy tour</th>
      <th>Galaxy History</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Introduction to Transcriptomics</td>
      <td><a href="https://galaxyproject.github.io/training-material/topics/transcriptomics/slides/introduction.html" target="_blank"><i class="fa fa-slideshare" aria-hidden="true"></i></a></td>
      <td> </td>
      <td> </td>
      <td> </td>
      <td> </td>
      <td> </td>
    </tr>
    <tr>
      <td>Plates, Batches, and Barcodes</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-plates-batches-barcodes/slides.html" target="_blank"><i class="fa fa-slideshare" aria-hidden="true"></i></a></td>
      <td> </td>
      <td> </td>
      <td> </td>
      <td> </td>
      <td> </td>
    </tr>
    <tr>
      <td>Understanding Barcodes</td>
      <td> </td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-umis/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/2573177" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td> </td>
      <td> </td>
      <td> </td>
    </tr>
    <tr>
      <td>Pre-processing of Single-Cell RNA Data</td>
      <td> </td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/2573175" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://singlecell.usegalaxy.eu/workflows/run?id=49073a24429d93d6" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a> <a href="https://singlecell.usegalaxy.eu/workflows/run?id=76a6330d5fc241c7" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td> </td>
      <td><a href="https://singlecell.usegalaxy.eu/u/mehmet-tekman/h/celseq2-single-batch-mm10" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a> <a href="https://singlecell.usegalaxy.eu/u/mehmet-tekman/h/celseq2-multi-batch-workflow" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>Single-Cell Quality Control with Scater</td>
      <td> </td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-scater-qc/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/3386291" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://singlecell.usegalaxy.eu/workflows/run?id=50f8693fac7e134b" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td> </td>
      <td> </td>
    </tr>
    <tr>
      <td>Downstream Single-cell RNA analysis with RaceID</td>
      <td> </td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-raceid/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/1511582" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://singlecell.usegalaxy.eu/workflows/run?id=2f8d9fb85242eca7" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td> </td>
      <td><a href="https://singlecell.usegalaxy.eu/u/mehmet-tekman/h/raceid-training-material" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>Clustering 3K PBMCs with ScanPy</td>
      <td> </td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/3581213" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://singlecell.usegalaxy.eu/workflows/run?id=921cab3e6faf30be" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td> </td>
      <td> </td>
    </tr>
  </tbody>
</table>

<h1 id="available-tools">Available tools</h1>

<p>In this section we list all tools that have been integrated in the the single cell workbench. The list is likely to grow as soon as further tools and workflows are contributed. To ease readability, we divided them into categories.</p>

<h2 id="preprocessing">Preprocessing</h2>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th style="text-align: right"> </th>
      <th style="text-align: center">Formats</th>
      <th style="text-align: center">QC</th>
      <th style="text-align: center">Demultiplexing</th>
      <th style="text-align: center">Mapping</th>
      <th style="text-align: center">Quantification</th>
      <th style="text-align: left">Description</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: right"><a href="#suites-mentioned">AnnData / Loom Suite</a></td>
      <td style="text-align: center">CSV LOOM H5</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: left">Import, Export, Modify, Inspect and Convert AnnData files. Converts between various different single-cell formats and permits various levels of inspection and manipulation. <a href="https://anndata.readthedocs.io/en/latest/">AnnData Documentation</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/crosscontamination_barcode_filter" target="_top">Cross Contamination Barcode Filter</a></td>
      <td style="text-align: center">CSV</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: left">QC tool for inspecting raw count matrices and determining the level of contamination between neighbouring sequencing wells</td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/fastp/fastp/" target="_top">fastp</a></td>
      <td style="text-align: center">FASTA/Q</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: left">Fast all-in-one preprocessing for FASTQ files <a href="https://doi.org/10.1101/274100" target="_blank">Shifu et al. 2018</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc" target="_top">FastQC</a> / <a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/" target="_top">MultiQC</a></td>
      <td style="text-align: center">FASTQ</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: left">A quality control tool for high throughput sequence data / MultiQC aggregates results from bioinformatics analyses into a single report <a href="https://doi.org/10.1093/bioinformatics/btw354" target="_blank">Ewels et al. 2016</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/lparsons/htseq_count/htseq_count" target="_top">htseq-count</a></td>
      <td style="text-align: center">BAM</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Y</td>
      <td style="text-align: left">Tool for counting reads in features <a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu638" target="_blank">Anders et al. 2015</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="#suites-mentioned">Je Suite</a></td>
      <td style="text-align: center">FASTA/Q BAM</td>
      <td style="text-align: center">Filtering</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Y</td>
      <td style="text-align: left">A versatile suite to handle multiplexed NGS libraries with unique molecular identifiers <a href="https://doi.org/10.1186/s12859-016-1284-2">Girardot at al. 2016</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="#suites-mentioned">UMI-tools Suite</a></td>
      <td style="text-align: center">FASTQ</td>
      <td style="text-align: center">Filtering</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Y</td>
      <td style="text-align: left">Includes utilities to count, deduplicate, extract, group, and whitelist Unique Molecular Identifiers (UMIs). Modelling sequencing errors in Unique Molecular Identifiers to improve quantification accuracy <a href="https://doi.org/10.1101/gr.209601.116" target="_blank">Smith et al. 2017</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star" target="_top">RNA STAR</a></td>
      <td style="text-align: center">FASTA/Q</td>
      <td style="text-align: center">Filtering</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: left">Rapid spliced aligner for RNA-seq data <a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537/STAR-ultrafast-universal-RNA-seq-aligner" target="_blank">Dobin et al. 2013</a>, performs single-cell quantification only when used with <em>featureCounts</em></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/rna_starsolo" target="_top">RNA STAR Solo</a></td>
      <td style="text-align: center">FASTA/Q</td>
      <td style="text-align: center">Filtering</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: left">Supports Droplet (Drop-seq, 10X) protocols, and can emulate CellRanger pipeline <a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537/STAR-ultrafast-universal-RNA-seq-aligner" target="_blank">Dobin et al. 2013</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/bowtie2/bowtie2" target="_top">Bowtie2</a></td>
      <td style="text-align: center">FASTA/Q</td>
      <td style="text-align: center">Filtering</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center"> </td>
      <td style="text-align: left">Fast and sensitive read alignment <a href="https://doi.org/10.1038/nmeth.1923" target="_blank">Langmead et al. 2012</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/hisat2/hisat2" target="_top">HISAT2</a></td>
      <td style="text-align: center">FASTA/Q</td>
      <td style="text-align: center">Filtering</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center"> </td>
      <td style="text-align: left">Hierarchical indexing for spliced alignment of transcripts <a href="https://doi.org/10.1038/nprot.2016.095" target="_blank">Pertea et al. 2016</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/sailfish/sailfish" target="_top">Sailfish</a></td>
      <td style="text-align: center">FASTA/Q BAM</td>
      <td style="text-align: center">Filtering</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: left">Rapid Alignment-free Quantification of Isoform Abundance <a href="http://dx.doi.org/10.1038/nbt.2862" target="_blank">Patro et al. 2014</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/alevin" target="_top">Alevin</a> / <a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/salmon/salmon" target="_top">Salmon</a></td>
      <td style="text-align: center">FASTA/Q BAM</td>
      <td style="text-align: center">Filtering</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Salmon</td>
      <td style="text-align: center">Salmon</td>
      <td style="text-align: left">Fast, accurate and bias-aware transcript quantification <a href="http://dx.doi.org/10.1038/nmeth.4197" target="_blank">Patro et al. 2017</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/scpipe/scpipe/" target="_top">scPipe</a></td>
      <td style="text-align: center">FASTQ BAM</td>
      <td style="text-align: center">Filtering</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">Subread</td>
      <td style="text-align: center">Subread</td>
      <td style="text-align: left">Preprocessing pipeline for single cell RNA-seq <a href="https://doi.org/10.1371/journal.pcbi.1006361" target="_blank">Tian et al. 2018</a>. Also performs downstream analysis via Scater and SCRAN</td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/featurecounts/featurecounts" target="_top">featureCounts</a></td>
      <td style="text-align: center">BAM</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Subread</td>
      <td style="text-align: center">Subread</td>
      <td style="text-align: left">Ultrafast and accurate read summarization program <a href="http://dx.doi.org/10.1093/bioinformatics/btt656" target="_blank">Liao et al. 2014</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/bwa/bwa_mem/" target="_top">BWA-MEM</a></td>
      <td style="text-align: center">FASTA/Q</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Y, not splice-aware</td>
      <td style="text-align: center"> </td>
      <td style="text-align: left">Software package for mapping low-divergent sequences against a large reference genome <a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank">Li and Durbin 2009</a>, <a href="https://doi.org/10.1093/bioinformatics/btp698" target="_blank">Li and Durbin 2010</a></td>
    </tr>
  </tbody>
</table>

<h2 id="downstream">Downstream</h2>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th style="text-align: right"> </th>
      <th style="text-align: center">Formats</th>
      <th style="text-align: center">Filtering</th>
      <th style="text-align: center">Normalisation</th>
      <th style="text-align: center">Batch / Confounder Removal</th>
      <th style="text-align: center">Clustering</th>
      <th style="text-align: center">Embedding</th>
      <th style="text-align: center">Lineage/Pseudotime</th>
      <th style="text-align: center">Classification / Marker</th>
      <th style="text-align: left">Description</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: right"><a href="#suites-mentioned">SC3 Suite</a></td>
      <td style="text-align: center">SCE</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Consensus K-means</td>
      <td style="text-align: center">PCA</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: left">Consensus clustering of single-cell RNA-Seq data <a href="https://doi.org/10.1038/nmeth.4236">Kiselev et al</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="#suites-mentioned">ScanPy Suite</a></td>
      <td style="text-align: center">AnnData CSV LOOM excel mtx umi-tools</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">CPM</td>
      <td style="text-align: center">Cell Cycle ComBat MNNCorrect BBKNN</td>
      <td style="text-align: center">PCA leiden louvain</td>
      <td style="text-align: center">PCA tSNE UMAP</td>
      <td style="text-align: center">dendrogram PAGA</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: left">The first scRNA pipeline written in Python with native 10X support <a href="https://doi.org/10.1186/s13059-017-1382-0">Wolf at al. 2018</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="#suites-mentioned">Scater Suite</a></td>
      <td style="text-align: center">SCE SC3</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">CPM FPKM TPM</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center">PCA</td>
      <td style="text-align: center">PCA tSNE UMAP PHATE diffmap</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: left">Pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R <a href="https://doi.org/10.1093/bioinformatics/btw777">McCarthy et al. 2017</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="#suites-mentioned">Seurat Suite</a></td>
      <td style="text-align: center">CSV 10X H5 Alevin</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">Log-Normalise Centered-log Relative-counts</td>
      <td style="text-align: center">SCTransform SNNAnchor</td>
      <td style="text-align: center">SharedNN</td>
      <td style="text-align: center">PCA tSNE UMAP</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Y</td>
      <td style="text-align: left">An R package designed for QC, analysis, and exploration of single-cell RNA-seq data <a href="https://doi.org/10.1038/nbt.3192">Satija et al. 2015</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="#suites-mentioned">RaceID Suite</a></td>
      <td style="text-align: center">CSV SCSeq</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">Library Size</td>
      <td style="text-align: center">Linear CCcorrect (Cell cycle)</td>
      <td style="text-align: center">k-means k-medians hclust</td>
      <td style="text-align: center">PCA tSNE UMAP Fruchterman-Rheingold</td>
      <td style="text-align: center">StemID FateID</td>
      <td style="text-align: center"> </td>
      <td style="text-align: left">Clustering using RaceID performs clustering, outlier detection, dimensional reduction <a href="https://doi.org/10.1038/nmeth.4662">Herman et al. 2018</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/table_compute/table_compute" target="_top">Table Compute</a></td>
      <td style="text-align: center">CSV</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">user-defined function</td>
      <td style="text-align: center">user-defined function</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: left">A tool to perform complex operations upon single or multiple tables using the pandas framework.</td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/deseq2/deseq2" target="_top">DESeq2</a></td>
      <td style="text-align: center">CSV htseq-count summarizedexperiment</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">DESeq2</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: left">Differential gene expression analysis based on the negative binomial distribution <a href="http://doi.org/10.1186/s13059-014-0550-8" target="_blank">Love et al. 2014</a></td>
    </tr>
    <tr>
      <td style="text-align: right"><a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/scpipe/scpipe/" target="_top">scPipe</a></td>
      <td style="text-align: center">CSV AnnData</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center">Y</td>
      <td style="text-align: center"> </td>
      <td style="text-align: center"> </td>
      <td style="text-align: left">CelSeq2, etc. Uses Scater and SCRAN for downstream <a href="https://doi.org/10.1371/journal.pcbi.1006361">Tian et al. 2018</a>, that also performs pre-processing.</td>
    </tr>
  </tbody>
</table>
<!-- |                                                                                                                                                                                                                                                            Monocle Suite: <b>Unknown Tool</b> <b>Unknown Tool</b> <b>Unknown Tool</b> <b>Unknown Tool</b> <b>Unknown Tool</b> <b>Unknown Tool</b> <b>Unknown Tool</b> <b>Unknown Tool</b> | CSV AnnData                          | Y         | Library Size                               | Y                                  | Louvain KNN GMM          | ICA tSNE L1-graph SimplePPT DDRTree | Y                  | Y                       | An analysis toolskit for single-cell RNA-seq that works well with 10X [Trapnell et al. 2014](https://doi.org/10.1038/nbt.2859)                                          | -->

<h2 id="suites-mentioned">Suites Mentioned</h2>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th style="text-align: right">Suite Name</th>
      <th style="text-align: left">Tools</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: right">AnnData / Loom</td>
      <td style="text-align: left">[Export](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/anndata_export" target="_top">anndata_export</a>) [Import](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/anndata_import" target="_top">anndata_import</a>) [Inspect](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/anndata_inspect" target="_top">anndata_inspect</a>) [Manipulate](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/anndata_manipulate" target="_top">anndata_manipulate</a>) [Modify Loom](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/modify_loom" target="_top">modify_loom</a>)</td>
    </tr>
    <tr>
      <td style="text-align: right">Je</td>
      <td style="text-align: left">[Clip](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/gbcs-embl-heidelberg/je_clip" target="_top">je_clip</a>) [Demultiplex](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/gbcs-embl-heidelberg/je_demultiplex" target="_top">je_demultiplex</a>) [Demultiplex Illumina](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/gbcs-embl-heidelberg/je_demultiplex_illu" target="_top">je_demultiplex_illu</a>) [Mark Dupes](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/gbcs-embl-heidelberg/je_markdupes" target="_top">je_markdupes</a>)</td>
    </tr>
    <tr>
      <td style="text-align: right">UMI-tools</td>
      <td style="text-align: left">[Count](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/umi_tools_count" target="_top">umi_tools_count</a>) [De-duplicate](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/umi_tools_dedup" target="_top">umi_tools_dedup</a>) [Extract](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/umi_tools_extract" target="_top">umi_tools_extract</a>) [Group](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/umi_tools_group" target="_top">umi_tools_group</a>) [Whitelist](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/umi_tools_whitelist" target="_top">umi_tools_whitelist</a>)</td>
    </tr>
    <tr>
      <td style="text-align: right">SC3</td>
      <td style="text-align: left">[Calc Consens](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/sc3_calc_consens/sc3_calc_consens" target="_top">sc3_calc_consens</a>) [Prepare](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/sc3_prepare/sc3_prepare" target="_top">sc3_prepare</a>) [Calc Tranfs](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/sc3_calc_transfs/sc3_calc_transfs" target="_top">sc3_calc_transfs</a>) [Calc_Biology](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/sc3_calc_biology/sc3_calc_biology" target="_top">sc3_calc_biology</a>) [Estimate K](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/sc3_estimate_k/sc3_estimate_k" target="_top">sc3_estimate_k</a>) [Calc Dists](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/sc3_calc_dists/sc3_calc_dists" target="_top">sc3_calc_dists</a>) [K-means](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/sc3_kmeans/sc3_kmeans" target="_top">sc3_kmeans</a>)</td>
    </tr>
    <tr>
      <td style="text-align: right">ScanPy</td>
      <td style="text-align: left">[Read 10X](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_read_10x/scanpy_read_10x" target="_top">scanpy_read_10x</a>) [Scale Data](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_scale_data/scanpy_scale_data" target="_top">scanpy_scale_data</a>) [Find Cluster](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_find_cluster/scanpy_find_cluster" target="_top">scanpy_find_cluster</a>)  [Run PCA](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_run_pca/scanpy_run_pca" target="_top">scanpy_run_pca</a>) [Run tSNE](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_run_tsne/scanpy_run_tsne" target="_top">scanpy_run_tsne</a>) [Run UMAP](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_run_umap/scanpy_run_umap" target="_top">scanpy_run_umap</a>) [Compute Graph](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_compute_graph/scanpy_compute_graph" target="_top">scanpy_compute_graph</a>) [Find Variable Genes](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_find_variable_genes/scanpy_find_variable_genes" target="_top">scanpy_find_variable_genes</a>) [Normalise Data](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_normalise_data/scanpy_normalise_data" target="_top">scanpy_normalise_data</a>) [Filter genes](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_filter_genes/scanpy_filter_genes" target="_top">scanpy_filter_genes</a>) [Find Markers](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_find_markers/scanpy_find_markers" target="_top">scanpy_find_markers</a>) [Filter Cells](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_filter_cells/scanpy_filter_cells" target="_top">scanpy_filter_cells</a>) [Parameter Iterator](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_parameter_iterator/scanpy_parameter_iterator" target="_top">scanpy_parameter_iterator</a>)</td>
    </tr>
    <tr>
      <td style="text-align: right">Scater</td>
      <td style="text-align: left">[Is Outlier](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scater_is_outlier/scater_is_outlier" target="_top">scater_is_outlier</a>) [Calculate CPM](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scater_calculate_cpm/scater_calculate_cpm" target="_top">scater_calculate_cpm</a>) [Calculate QC Metrics](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scater_calculate_qc_metrics/scater_calculate_qc_metrics" target="_top">scater_calculate_qc_metrics</a>) [Read 10X Results](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scater_read_10x_results/scater_read_10x_results" target="_top">scater_read_10x_results</a>) [Filter](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scater_filter/scater_filter" target="_top">scater_filter</a>) [Normalize](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scater_normalize/scater_normalize" target="_top">scater_normalize</a>)</td>
    </tr>
    <tr>
      <td style="text-align: right">Seurat</td>
      <td style="text-align: left">[Read 10X](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_read10x/seurat_read10x" target="_top">seurat_read10x</a>) [Run PCA](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_run_pca/seurat_run_pca" target="_top">seurat_run_pca</a>) [Scale Data](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_scale_data/seurat_scale_data" target="_top">seurat_scale_data</a>) [Filter Cells](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_filter_cells/seurat_filter_cells" target="_top">seurat_filter_cells</a>) [Find Markers](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_find_markers/seurat_find_markers" target="_top">seurat_find_markers</a>) [Dim. Plot](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_dim_plot/seurat_dim_plot" target="_top">seurat_dim_plot</a>) [Normalise Data](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_normalise_data/seurat_normalise_data" target="_top">seurat_normalise_data</a>) [Create Seurat Object](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_create_seurat_object/seurat_create_seurat_object" target="_top">seurat_create_seurat_object</a>) [Export CellBrowser](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_export_cellbrowser/seurat_export_cellbrowser" target="_top">seurat_export_cellbrowser</a>) [Run tSNE](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_run_tsne/seurat_run_tsne" target="_top">seurat_run_tsne</a>) [Find Variable Genes](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_find_variable_genes/seurat_find_variable_genes" target="_top">seurat_find_variable_genes</a>) [Find Clusters](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_find_clusters/seurat_find_clusters" target="_top">seurat_find_clusters</a>)</td>
    </tr>
    <tr>
      <td style="text-align: right">RaceID</td>
      <td style="text-align: left">[Clustering](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/raceid_clustering" target="_top">raceid_clustering</a>) [Filt.Norm.Conf](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/raceid_filtnormconf/" target="_top">raceid_filtnormconf</a>) [Trajectory](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/raceid_trajectory/" target="_top">raceid_trajectory</a>) [Inspect Clusters](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/raceid_inspectclusters/" target="_top">raceid_inspectclusters</a>) [Inspect Trajectory](<a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/raceid_inspecttrajectory" target="_top">raceid_inspecttrajectory</a>)</td>
    </tr>
  </tbody>
</table>

<!-- BEGIN: The following tools were left out: --->
<!-- <a href="https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/star_fusion/star_fusion" target="_top">STAR-fusion</a> | Fast fusion gene finder | [Haas et al. 2017](https://www.biorxiv.org/content/early/2017/03/24/120295){:target="_blank"} -->
<!-- ## Utilities -->
<!-- SAMtools | Utilities for manipulating alignments in the SAM format | [Heng et al. 2009](https://doi.org/10.1093/bioinformatics/btp352){:target="_blank"} -->
<!-- BEDTools | Utilities for genome arithmetic | [Quinlan and Hall 2010](https://doi.org/10.1093/bioinformatics/btq033){:target="_blank"} -->
<!-- deepTools | Tools for exploring deep-sequencing data | [Ramirez et al. 2014](https://doi.org/10.1093/nar/gku365){:target="_blank"}, [Ramirez et al. 2016](https://doi.org/10.1093/nar/gkw257){:target="_blank"} -->
<!-- END --->

<h1 id="contributors">Contributors</h1>

<ul>
  <li><a href="https://github.com/bebatut">Bérénice Batut</a></li>
  <li><a href="https://github.com/mtekman">Mehmet Tekman</a></li>
  <li><a href="https://github.com/astrovsky01">Alex Ostrovsky</a></li>
  <li><a href="https://github.com/mblue9">Maria Doyle</a></li>
  <li><a href="https://github.com/pcm32">Pablo Moreno</a></li>
  <li><a href="https://github.com/nh3">Ni Huang</a></li>
  <li><a href="https://github.com/pinin4fjords">Jonathan Manning</a></li>
  <li><a href="https://github.com/drosofff">Christophe Antoniewski</a></li>
  <li><a href="https://github.com/bellenger-l">Lea Bellenger</a></li>
  <li><a href="https://github.com/scholtalbers">Jelle Scholtalbers</a></li>
  <li><a href="https://github.com/cgirardot">Charles Girardot</a></li>
  <li><a href="https://github.com/nsoranzo">Nicola Soranzo</a></li>
  <li><a href="https://github.com/ethering">Graham Etherington</a></li>
  <li><a href="https://github.com/fidelram">Fidel Ramirez</a></li>
  <li><a href="https://github.com/blankenberg">Daniel Blankenberg</a></li>
  <li><a href="https://github.com/jxtx">James Taylor</a></li>
  <li><a href="https://github.com/bgruening">Bjoern Gruening</a></li>
</ul>



<h2>Our Data Policy</h2>

<h3>Registered Users</h3>

User data on UseGalaxy.eu (i.e. datasets, histories) will be available as long
as they are not deleted by the user. Once marked as deleted the datasets will
be permanently removed within 14 days. If the user "purges" the dataset in the
Galaxy, it will be removed immediately, permanently.

An extended quota can be <a href="https://docs.google.com/forms/d/e/1FAIpQLSf9w2MOS6KOlu9XdhRSDqWnCDkzoVBqHJ3zH_My4p8D8ZgkIQ/viewform" target="_blank">requested</a>
for a limited time period in special cases.

<h3>Unregistered Users</h3>

Processed data will only be accessible during one browser session, using a
cookie to identify your data. This cookie is not used for any other purposes
(e.g. tracking or analytics.)

If UseGalaxy.eu service is not accessed for 90 days, those datasets will be
permanently deleted.

<h3>FTP Data</h3>
Any user data uploaded to our FTP server should be imported into Galaxy as soon
as possible. Data left in FTP folders for more than 3 months, will be deleted.

<h3>GDPR Compliance</h3>

The Galaxy service complies with the EU General Data Protection Regulation
(GDPR). You can read more about this on our
<a href="https://usegalaxy.eu/terms/">Terms and Conditions</a>.

<div>
<iframe style="border: 0px" width="100%" height="150px" src="https://stats.galaxyproject.eu/dashboard-solo/db/jobs-dashboard?panelId=1&refresh=1m&orgId=1" ></iframe>
</div>
<div>
<!--<iframe style="border: 0px" width="33%" height="100px" src="https://stats.galaxyproject.eu/d-solo/000000034/jobs-dashboard?orgId=1&refresh=1m&panelId=3" ></iframe>-->
</div>

<div class="row">

    <section class="section-content">
        <div class="col-md-12">
        </div>
    </section>
</div>

</div>

        </div>
      </div>
    </div>
    






<script>
  ((window.gitter = {}).chat = {}).options = {
  room: 'usegalaxy-eu/Lobby'
  };
</script>
<script src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer></script>


<footer class="navbar-default">
    <div class="container">
        <div class="row">
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <p>UseGalaxy.eu is maintained largely by the <a href="/freiburg/">Freiburg Galaxy Team</a> but also collectively by groups and individuals from across Europe. All of the members sites in this repository contribute to the European Galaxy Project.
All content on this site is available under <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0-1.0</a>, unless otherwise specified.</p>

            </div>
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <ul class="contact-info">
                    <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu/website/tree/master/index-singlecell.md">Edit this page on GitHub</a></li>
                    
                      <li><i class="fa fa-envelope"></i><a href="mailto:contact@usegalaxy.eu">contact@usegalaxy.eu</a></li>
                    
                    
                      <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu">usegalaxy-eu</a></li>
                    
                    
                      <li><i class="fa fa-twitter"></i><a href="https://twitter.com/galaxyproject">galaxyproject</a></li>
                    
                      <li><i class="fa fa-rss"></i>Subscribe <a href="/feed.xml">via RSS (UseGalaxy.eu Feed)</a></li>
                </ul>
            </div>
        </div>
    </div>
</footer>

  </body>
</html>
