<!DOCTYPE html>
<html lang="en">
  <head>
  <title>Galaxy Europe</title>
  <meta property="og:title" content="" />
  <meta property="og:description" content="" />
  <meta property="og:image" content="/assets/media/galaxy-eu-logo.512.png" />
  <meta name="description" content="The European Galaxy Instance">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">




  <link rel="canonical" href="https://galaxyproject.eu/index-graphclust.html">
  <link rel="shortcut icon" href="/assets/media/galaxy-eu-logo.64.png" type="image/x-icon" />
  <link rel="alternate" type="application/rss+xml" title="Galaxy Europe" href="/feed.xml">

  <link href="/assets/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
  <script src="/assets/js/jquery-3.2.1.slim.min.js" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" crossorigin="anonymous"></script>
  <script src="/assets/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
</head>

  <body>
    <div id="wrap">
      <div id="main">
        <div class="container" id="maincontainer">
          <div class="home">







  <p><br /></p>

<p>Welcome to <strong>Galaxy GraphClust</strong> – a webserver to process and analyse structural clustering of RNA secondary structures.</p>

<p>Galaxy-GraphClust is a web-based workflow for structural clustering of RNA secondary structures developed as an instance of <a href="http://www.bioinf.uni-freiburg.de/Software/GraphClust">GraphClust</a> Perl pipeline inside the Galaxy framework. It consists of a set of integrated Galaxy tools and different flavors of clustering workflows built upon these tools.</p>

<ol id="markdown-toc">
  <li><a href="#get-started" id="markdown-toc-get-started">Get started</a></li>
  <li><a href="#graphclust-pipeline-overview" id="markdown-toc-graphclust-pipeline-overview">GraphClust pipeline overview</a></li>
  <li><a href="#support--bug-reports" id="markdown-toc-support--bug-reports">Support &amp; Bug Reports</a></li>
  <li><a href="#references" id="markdown-toc-references">References</a></li>
</ol>

<h1 id="get-started">Get started</h1>

<p>Are you new to Galaxy, or returning after a long time, and looking for help to get started? Take <a href="https://metagenomics.usegalaxy.eu/tours/core.galaxy_ui" target="_top">a guided tour</a> through Galaxy’s user interface.</p>

<h3 id="video-tutorial">Video tutorial</h3>

<p><a href="https://www.youtube.com/watch?v=fJ6tUt_6uas">This video tutorial</a> can be helpful to get a visually comprehensive introduction on setting-up and running Galaxy-GraphClust.</p>

<p><a href="https://www.youtube.com/watch?v=fJ6tUt_6uas" target="_top"><img src="/assets/media/graphclust_video-thumbnail.png" alt="IMAGE ALT TEXT HERE" /></a></p>

<h3 id="interactive-tours">Interactive tours</h3>
<p>Interactive Tours are available for Galaxy and Galaxy-GraphClust. To run the tours please on top panel go to <strong>Help→Interactive Tours</strong> and click on one of the tours prefixed <strong>GraphClust workflow</strong>. You can check the other tours for a more general introduction to the Galaxy interface.</p>

<h3 id="import-or-upload-a-workflow">Import or upload a workflow</h3>

<p>To import or upload an existing workflow, on the top panel go to <strong>Workflow</strong> menu. On top right side of the screen click on <strong>Upload or import workflow</strong> button. You can either upload workflow from your local system or by providing the URL of the workflow. To have an access to workflow menu you must be logged in. You can download workflows from the following links</p>

<ul>
  <li>MotifFinder: <a href="https://graphclust.usegalaxy.eu/u/graphclust2/w/graphclust2--motiffinder">GraphClust-MotifFinder</a></li>
  <li>Workflow main: <a href="https://graphclust.usegalaxy.eu/u/graphclust2/w/graphclust2--main-1r">GraphClust_1r</a></li>
  <li>Workflow main, preconfigured for two rounds : <a href="https://graphclust.usegalaxy.eu/u/graphclust2/w/graphclust2--main-2r">GraphClust_2r</a></li>
</ul>

<h1 id="graphclust-pipeline-overview">GraphClust pipeline overview</h1>

<p>GraphClust pipeline for clustering similar RNA sequences together is a multi-step rather complex pipeline. Overall it consists of three major phases: a) sequence based pre-clustering b) encoding predicted RNA structures as graph features c) iterative fast candidate clustering then refinement</p>

<!-- ![GraphClust pipeline overview (Heyne et al. 2012)](./assets/img/graphclust_pipeline.png)

*GraphClust pipeline overview (Heyne et al. 2012)*
 -->
<p><img src="https://github.com/BackofenLab/GraphClust-2/blob/master/assets/img/figure-pipeline_zigzag.png" alt="GraphClust-2 workflow overview" /></p>

<p>Below is a coarse-grained correspondence list of GraphClust2 tool names with each step:</p>

<table class="table table-striped">
  <thead>
    <tr>
      <th style="text-align: center">Stage</th>
      <th style="text-align: left">Galaxy Tool Name</th>
      <th style="text-align: left">Description</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center">1</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_preprocessing/preproc/0.5" target="_top">Preprocessing</a></td>
      <td style="text-align: left">Input preprocessing (fragmentation)</td>
    </tr>
    <tr>
      <td style="text-align: center">2</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_fasta_to_gspan/gspan/0.4" target="_top">fasta_to_gspan</a></td>
      <td style="text-align: left">Generation of structures via RNAshapes and conversion into graphs</td>
    </tr>
    <tr>
      <td style="text-align: center">3</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_nspdk/nspdk_sparse/9.2.3" target="_top">NSPDK_sparseVect</a></td>
      <td style="text-align: left">Generation of graph features via NSPDK</td>
    </tr>
    <tr>
      <td style="text-align: center">4</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_nspdk/NSPDK_candidateClust/9.2.3" target="_top">NSPDK_candidateClusters</a></td>
      <td style="text-align: left">min-hash based clustering of all feature vectors, output top dense candidate clusters</td>
    </tr>
    <tr>
      <td style="text-align: center">5</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_prepocessing_for_mlocarna/preMloc/0.4" target="_top">PGMA_locarna</a>,<a href="https://graphclust.usegalaxy.eu/tool_runner?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_mlocarna/locarna_best_subtree/0.4" target="_top">locarna</a>, <a href="https://graphclust.usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_cmfinder/cmFinder/0.4" target="_top">CMfinder</a></td>
      <td style="text-align: left">Locarna based clustering of each candidate cluster, all-vs-all pairwise alignments, create multiple alignments along guide tree, select best subtree, and refine alignment.</td>
    </tr>
    <tr>
      <td style="text-align: center">6</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/infernal/infernal_cmbuild/1.1.0.2" target="_top">Build covariance models</a></td>
      <td style="text-align: left">create candidate model</td>
    </tr>
    <tr>
      <td style="text-align: center">7</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/infernal/infernal_cmsearch/1.1.0.2" target="_top">Search covariance models</a></td>
      <td style="text-align: left">Scan full input sequences with Infernal’s cmsearch to find missing cluster members</td>
    </tr>
    <tr>
      <td style="text-align: center">8,9</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_postprocessing/glob_report/0.5">Report results</a> and <a href="https://graphclust.usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam%2Fgraphclust_aggregate_alignments/graphclust_aggregate_alignments/0.1">conservation evaluations</a></td>
      <td style="text-align: left">Collect final clusters and create example alignments of top cluster members</td>
    </tr>
  </tbody>
</table>

<h3 id="input">Input:</h3>
<p>The input to the workflow is a set of putative RNA sequences in FASTA format. Inside the <code class="highlighter-rouge">sample_data</code> directory you can find examples of the input format. In this case the data is from a benchmark set based on Rfam 12.0 and additionally is optionally labeled with reference Rfam family members.</p>

<h3 id="configuring-the-workflows">Configuring the workflows:</h3>
<p>Please proceed with the interactive tour named <code class="highlighter-rouge">GraphClust workflow step by step</code>, available under <code class="highlighter-rouge">Help-&gt;Interactive Tours</code>
Check FAQs for understanding the frequently important parameters.</p>

<h3 id="output">Output:</h3>
<p>The output contains the predicted clusters, where similar putative input RNA sequences form a cluster. Additionally overall status of the clusters and the matching of cluster elements is reported for each cluster.</p>

<p>Please check the interactive tours and GraphClust <a href="http://www.bioinf.uni-freiburg.de/Software/GraphClust/README">README</a> for more information about the reported info and files.</p>

<h1 id="support--bug-reports">Support &amp; Bug Reports</h1>

<p>You can file an <a href="https://github.com/BackofenLab/docker-galaxy-graphclust/issues">github issue</a> or ask us on the <a href="http://lists.bx.psu.edu/listinfo/galaxy-dev">Galaxy development list</a>.</p>

<h1 id="references">References</h1>

<ol class="bibliography"></ol>


<h2>Our Data Policy</h2>

<h3>Registered Users</h3>

User data on UseGalaxy.eu (i.e. datasets, histories) will be available as long
as they are not deleted by the user. Once marked as deleted the datasets will
be permanently removed within 14 days. If the user "purges" the dataset in the
Galaxy, it will be removed immediately, permanently.

An extended quota can be <a href="https://docs.google.com/forms/d/e/1FAIpQLSf9w2MOS6KOlu9XdhRSDqWnCDkzoVBqHJ3zH_My4p8D8ZgkIQ/viewform" target="_blank">requested</a>
for a limited time period in special cases.

<h3>Unregistered Users</h3>

Processed data will only be accessible during one browser session, using a
cookie to identify your data. This cookie is not used for any other purposes
(e.g. tracking or analytics.)

If UseGalaxy.eu service is not accessed for 90 days, those datasets will be
permanently deleted.

<h3>GDPR Compliance</h3>

The Galaxy service will be compliant to the EU General Data Protection
Regulation starting May, 25th 2018. You can read more about this on our GDPR
page: <a href="https://usegalaxy.eu/terms/">usegalaxy.eu/terms</a>

<iframe style="border: 0px" width="100%" height="150px" src="https://stats.galaxyproject.eu/dashboard-solo/db/jobs-dashboard?panelId=1&orgId=1" ></iframe>

<div class="row">

    <section class="section-content">
        <div class="col-md-12">
        </div>
    </section>
</div>

</div>

        </div>
      </div>
    </div>
    





<footer class="navbar-default">
    <div class="container">
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <p>UseGalaxy.eu is maintained largely by the <a href="/freiburg/">Freiburg Galaxy Team</a> but also collectively by groups and individuals from across Europe. All of the members sites in this repository contribute to the European Galaxy Project.
All content on this site is available under <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0-1.0</a>, unless otherwise specified.</p>

            </div>
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <ul class="contact-info">
                    
                      <li><i class="fa fa-envelope"></i><a href="mailto:contact@usegalaxy.eu">contact@usegalaxy.eu</a></li>
                    
                    
                      <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu">usegalaxy-eu</a></li>
                    
                    
                      <li><i class="fa fa-twitter"></i><a href="https://twitter.com/galaxyproject">galaxyproject</a></li>
                    
                      <li><i class="fa fa-rss"></i>Subscribe <a href="/feed.xml">via RSS (UseGalaxy.eu Feed)</a></li>
                </ul>
            </div>
        </div>
    </div>
</footer>

  </body>
</html>
